src/SFA.DAS.ApplyService.InternalApi/Controllers/FinancialController.cs

```diff
@@ -40,6 +41,13 @@ public async Task<ActionResult> ClosedApplications()
             return Ok(applications);
         }
 
+        [HttpPost("/Financial/Grade/{applicationId}")]
+        public async Task<IActionResult> RecordGrade(Guid applicationId, [FromBody] FinancialReviewDetails financialReviewDetails)
+        {
+            await _mediator.Send(new RecordGradeRequest(applicationId, financialReviewDetails));
+            return Ok();
+        }
+
```

Comment:
ahhh just noticed this. If I remember correct this is what UpdateGrade would have done.

As a minimum can you change the Route to: [HttpPost("/Financial/{applicationId}/Grade")]
This will make it consistent with all the other end points

Reply:
Done